scripts/globals/mobskills/stygian_vapor.lua

```diff
@@ -14,8 +14,8 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    skill:setMsg(xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.PLAGUE, 5, 0, 120))
-
+    local typeEffect = xi.effect.PLAGUE
```

Comment:
Same here, return `xi.effect.PLAGUE`

Reply:
![KO](https://user-images.githubusercontent.com/16403542/210649394-2c260929-66e3-4957-9d5e-0b43d62f84f7.JPG)
![para](https://user-images.githubusercontent.com/16403542/210649406-3bca9f77-203e-4e19-8cfc-cf1140ede020.JPG)

Reply:
Does in line 18